app/upgrades/v11/upgrades.go

```diff
@@ -0,0 +1,27 @@
+package v11
+
+import (
+	"context"
+
+	upgradetypes "cosmossdk.io/x/upgrade/types"
+	"github.com/cosmos/cosmos-sdk/types/module"
+
+	"github.com/milkyway-labs/milkyway/v10/app/keepers"
+)
+
+func CreateUpgradeHandler(
+	mm *module.Manager,
+	configuration module.Configurator,
+	keepers *keepers.AppKeepers,
+) upgradetypes.UpgradeHandler {
+	return func(ctx context.Context, plan upgradetypes.Plan, fromVM module.VersionMap) (module.VersionMap, error) {
+		vm, err := mm.RunMigrations(ctx, configuration, fromVM)
+		if err != nil {
+			return nil, err
+		}
+
+		// TODO: do we have to set default params for the new modules?
```

Comment:
I think we can set the ones provided by the modules. Usually there's a `types.DefaultParams()` method for each module that returns the default params and is used in the genesis and tests.

Reply:
Actually I just realized the modules don't have parameters. x/core doesn't have `Params` struct at all and x/warp has one but it's empty:
https://github.com/bcp-innovations/hyperlane-cosmos/blob/v1.0.0-rc0/x/warp/types/types.pb.go#L62-L64